internal/bminventory/inventory.go

```diff
@@ -2719,6 +2722,12 @@ func (b *bareMetalInventory) uploadLogs(ctx context.Context, params installer.Up
 		return common.NewApiError(http.StatusInternalServerError, err)
 	}
 
+	err = b.clusterApi.SetUploadControllerLogsAt(ctx, currentCluster, b.db)
```

Comment:
i think u need to validate log type here.

Reply:
You are right! updating.

Reply:
Will do that as well.